### DISCUSSION TITLE
Can I use NoiseModelling to compute noise sources other than traffic?

### DISCUSSION THREAD

**Discussion by engina** [Universite-Gustave-Eiffel/NoiseModelling#345] [Q&A] [answered]
Hello,

I wonder if I can abuse NoiseModelling infrastructure to compute the accumulation of mechanical (i.e. HVAC units) noises in a complex topological terrain.

These are points sources as opposed to line sources such as traffic but the principles should be similar.

Both of these calculations must rely heavily of rather accurate modelling of diffractions for buildings. 

I could model the terrain and structures and define the sky box as 100% absorbant and run on classical geometrical acoustical simulation softwares but I doubt their diffraction model can handle outdoor scenarios like this.

So, basically I'd need.

1. Topology (slopes) must be taken into account. The terrain is pretty much like an amphitheater so noise sources at different locations would contribute different amount of noise into the surface of interest.
2. Structural extrusions from the ground must affect (reflect) the sound propagation. Meaning, if a noise source if placed next to a building, the building must reflect its noise.
3. By nature of this computations, diffractions are essential.

Any pointers?

**Comment by nicolas-f** [comment, marked as answer]
Hi,

Yes its easy

You have to use the WPS block named Noise_Level_From_Source

The sound source table must have the following columns:

The table shall contain :
- PK : an identifier. It shall be a primary key (INTEGER, PRIMARY KEY).
- THE_GEOM : the 3D geometry of the sources (POINT, MULTIPOINT, LINESTRING, MULTILINESTRING). According to CNOSSOS-EU, you need to set a height of 0.05 m for a road traffic emission.
- LWD63, LWD125, LWD250, LWD500, LWD1000, LWD2000, LWD4000, LWD8000 : 8 columns giving the day emission sound level for each octave band (FLOAT).
- LWE* : 8 columns giving the evening emission sound level for each octave band (FLOAT).
- LWN* : 8 columns giving the night emission sound level for each octave band (FLOAT).

You have also the Digital Elevation Model table.

The table shall contain :
- THE_GEOM : the 3D geometry of the sources (POINT, MULTIPOINT).

So for a HVAC unit, you must use POINT instead of LINESTRING.

NoiseModelling does not have "skybox", it models only the ground and buildings

If you have large grass/snow/forest areas you should also use "Ground absorption" table that will contain the areas polygons.

You can enable vertical and horizontal diffraction on buildings edges. You can also enable reflection on building facades.

You should first follow the "Get started" and "NoiseMap from OSM tutorials" https://noisemodelling.readthedocs.io

best regards,

-Nicolas

**Reply by engina** [reply]
Hello Nicolas,

Indeed, Noise_Level_From_Source got my hopes up at first, but in the description (tooltip) it says road emission table, so I was skeptical if that was the right option.

I've completed the first tutorial. Now, I'm trying to follow the OSM tutorial. But I've noticed OSM doesn't export iso curves? How can be terrain taken into account if that's the case?

Note that I'm very new to maps. Also, I'm having trouble SRID, for Mugla, Turkey, I've come up with 7834 but not sure. I think these are just related to project corrections? And if it's a small region (<100,000m2)  I'm working on, it wouldn't make much of a difference maybe?

I'd be happy to see more pointers if you have some.

Greetings from Istanbul.

Engin

**Reply by pierromond** [reply]
To do :) #207 #137
